tiny_dnn/layers/max_pooling_layer.h

```diff
@@ -71,6 +78,27 @@ class max_pooling_layer : public layer {
                         (in_height == 1 ? 1 : pooling_size),
                         stride,
                         stride,
+                        ceil_mode,
+                        padding::valid,
+                        backend_type) {}
+
+  max_pooling_layer(size_t in_width,
+                    size_t in_height,
+                    size_t in_channels,
+                    size_t pooling_size_x,
+                    size_t pooling_size_y,
+                    size_t stride_x,
+                    size_t stride_y,
+                    padding pad_type             = padding::valid,
```

Comment:
The same question. https://github.com/tiny-dnn/tiny-dnn/pull/995/files#r226743476
It looks like only this constructor doesn't have `ceil_mode` parameter.